### PR TITLE
[TASK] Tackle shellcheck issues

### PR DESCRIPTION
**Pull request by cumuru**
Fixed or suppressed shellcheck issues to start with a passing pipeline for shellcheck

### CODE REVIEW

File: Build/Scripts/runTests.sh [hunk 1]
```diff
@@ -491,12 +493,12 @@ case ${TEST_SUITE} in
         cleanTestFiles
         ;;
     composer)
-        COMMAND="composer $@"
+        COMMAND="composer $*"
```
Comment (oliverklee) [inline Build/Scripts/runTests.sh:496]: This breaks this command:

```bash
./Build/Scripts/runTests.sh -s composer install -W
```

File: Build/Scripts/runTests.sh [hunk 2]
```diff
@@ -491,12 +493,12 @@ case ${TEST_SUITE} in
...
         ${CONTAINER_BIN} run ${CONTAINER_COMMON_PARAMS} --name composer-command-${SUFFIX} -e COMPOSER_CACHE_DIR=.cache/composer -e COMPOSER_ROOT_VERSION=${COMPOSER_ROOT_VERSION} ${IMAGE_PHP} /bin/sh -c "${COMMAND}"
         SUITE_EXIT_CODE=$?
         ;;
     composerInstall)
-        COMMAND="composer install $@"
+        COMMAND="composer install $*"
```
Comment (oliverklee) [inline Build/Scripts/runTests.sh:501]: This breaks:

```bash
./Build/Scripts/runTests.sh -s composerInstall -W
```

**Review by DanielSiepmann — APPROVED** [review]

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: Build/Scripts/runTests.sh (hunk 2 above)
Comment (cumuru) [reply Build/Scripts/runTests.sh:501]: It broke before and it breaks with TYPO3 core‘s `COMMAND=(composer install "$@")` solution. :shrug:

**Comment by oliverklee** [comment]
These are the commands that work in `main` and that should continue to work:

```bash
./Build/Scripts/runTests.sh -b podman -s composer update --with-dependencies
./Build/Scripts/runTests.sh -b docker -s composer update --with-dependencies
./Build/Scripts/runTests.sh -b podman -s composer update -W
./Build/Scripts/runTests.sh -b docker -s composer update -W
./Build/Scripts/runTests.sh -b podman -s composer install --no-progress
./Build/Scripts/runTests.sh -b docker -s composer install --no-progress
./Build/Scripts/runTests.sh -b podman -s composer install -q
./Build/Scripts/runTests.sh -b docker -s composer install -q
./Build/Scripts/runTests.sh -b podman -s composerInstall
./Build/Scripts/runTests.sh -b docker -s composerInstall
```

These commands do not work (yet), and we should tackle that in a separate PR:
```
./Build/Scripts/runTests.sh -b podman -s composerInstall --no-progress
./Build/Scripts/runTests.sh -b docker -s composerInstall --no-progress
./Build/Scripts/runTests.sh -b podman -s composerInstall -q
./Build/Scripts/runTests.sh -b docker -s composerInstall -q
```

**Review by oliverklee — APPROVED** [review]

**Pull request merged**